Dockerfile

```
@@ -38,6 +38,6 @@ COPY --from=builder /usr/local/bin /usr/local/bin
 COPY --from=builder /usr/local/lib /usr/local/lib
 COPY --from=builder /etc/kubernetes/node-feature-discovery /etc/kubernetes/node-feature-discovery
 RUN ldconfig
-COPY --from=builder /go/bin/node-feature-discovery /usr/bin/node-feature-discovery
+COPY --from=builder /go/bin/* /usr/bin/
```

```suggestion
COPY --from=builder /go/bin/nfd-* /usr/bin/
```
We need only the nfd-master and nfd-client binaries, other binaries shouldn't be copied into the final image. On my build systems there are other files in this directory.

---

You're right, thanks for pointing this out. Fixed!